css/style.css

```diff
@@ -865,8 +879,8 @@ footer nav {
   }
   /* footer styles */
 
-  /* adjusting nav elements in footer */
-  footer nav {
-    flex-direction: column;
+  /* Land Acknowledge button placement */
```

Comment:
does it look to anyone else like both la buttons are visible? is this because i didn't pull some other code with the media query hiding the desktop button? otherwise yay charli!!!!

Reply:
looked like that on mine before i pulled